src/loaders/svg/tvgSvgLoaderCommon.h

```diff
@@ -260,6 +260,8 @@ struct SvgDocNode
     float vy;
     float vw;
     float vh;
+    float wScalingFactor;
```

Comment:
You are using different names (sw, sh / wScalingFactor, hScalingFactor) for the same purpose. Please maintain consistency by using sw and sh.

Reply:
renamed